README.md

```diff
@@ -54,9 +79,82 @@ about "Cannot resolve module 'source-map-loader'" due to shortcomings in webpack
 Deployment
 ==========
 
+Configure the app by modifying the `config.json` file to the correct values:
+
+1. default_hs_url is for the home server url (could be http://your.server.ip:8008 if vector and synapse are on the same machine),
+2. default_is_url is for the identity server (could be empty but if you let it empty, email notifications and email login may not work).
+
+
 Just run `npm run build` and then mount the `vector` directory on your webserver to
 actually serve up the app, which is entirely static content.
 
+The port 8008 is used for http and 8448 for https. Set `http://your.server.ip:8008` or `https://your.server.ip:8448`
```

Comment:
This line is talking about synapse but everything else here is talking about vector. It would probably make more sense if this was made clear that this was talking about the default synapse ports and moved above the previous line to be next to the paragraph about config.json?